Modules/Media/Assets/js/components/MediaList.vue

```diff
@@ -4,8 +4,6 @@
             <div class="sc-table">
                 <div class="el-row">
                     <div class="title">
-                        <h4 v-if="singleModal">{{ trans('media.choose file') }}</h4>
-                        <h3 v-else>{{ trans('media.title.media') }}</h3>
```

Comment:
This H3 will no longer exist. It seems like the MediaList component is only used by the single/multiple file picker, which both use the `singleModal` attribute anyway